pkg/test/resourcefixture/testdata/basic/gkebackup/v1alpha1/gkebackupbackupplan/gkebackupbackupplan-minimal/_http.log

```diff
@@ -320,6 +320,18 @@ User-Agent: google-api-go-client/0.5 kcc/${kccVersion} (+https://github.com/Goog
     "binaryAuthorization": {
       "enabled": false
     },
+    "controlPlaneEndpointsConfig": {
+      "dnsEndpointConfig": {
+        "allowExternalTraffic": false
```

Comment:
A little worried that this may be a clobber on upgrade change. I'm assuming "allowExternalTraffic": false is the default. However if the customer manually over-rode this behavior, we may be clobbering that over-ride.

Reply:
IIRC, externally-managed fields are supported by default which means the real value will be used for reconciliation. Will double check.